cmake/EthCompilerSettings.cmake

```diff
@@ -89,9 +89,17 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MA
 
 	# Additional GCC-specific compiler settings.
 	if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
-		# Check that we've got GCC 8.0 or newer.
-		if (NOT (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 8.0))
-			message(FATAL_ERROR "${PROJECT_NAME} requires g++ 8.0 or greater.")
+		# Check that we've got GCC 12.0 or newer.
```

Comment:
```suggestion
		# Check that we've got GCC 11.0 or newer.
```

Reply:
oh yeah, i missed that, thanks! I wonder if these comments are even needed. It is pretty obvious what is happening. Oh well, fixed. :smile: